### PR TITLE
Reminder titles now link to each individual reminder

### LINKED ISSUE #3: See an individual reminder
The list of reminders should only display Titles.

When a user clicks on one of the existing titles, they are taken to an individual reminders route (ie: `'/reminders/1'`).

**Hints**:

* Ember convention uses `:model_id` as the dynamic path, ie: '/reminders/:reminder_id'.
* You can find help [here](https://guides.emberjs.com/v2.12.0/routing/specifying-a-routes-model/#toc_dynamic-models)
* Run `ember g route reminders/reminder`
* The user should still see all of the reminders from your previous feature. The specific reminder they are looking at should now be rendered in the `outlet` of the `reminders.hbs` template.
* There should be some CSS for the `.active` class.
**Hot Tip**: There is an existing acceptance test for this feature.

### PR DESCRIPTION
**Pull request by Mickyfen17**
Add nested reminder:id route within reminders
Change reminder titles on home page to link to individual reminders
Add reminder HTML to display each reminder upon title click

## Purpose

For user to be able to click each reminder title and have the details display below

close #3 

## Approach

[links](https://guides.emberjs.com/v2.12.0/templates/links/)
[route](https://guides.emberjs.com/v2.12.0/routing/defining-your-routes/)

### Learning

Nesting routes and adding links to html elements

### Test coverage 

To click on first title link, for the URL to change and to have the correct reminder display

### Follow-up tasks

Issue #4

### CODE REVIEW

**Comment by Mickyfen17** [comment]
@Tman22 @martensonbj

File: tests/unit/routes/reminders/reminder-test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
```
Comment (martensonbj) [inline tests/unit/routes/reminders/reminder-test.js]: Don't need this file

**Review by martensonbj — CHANGES_REQUESTED** [review]
Just need to delete an extra test file

File: tests/unit/routes/reminders/reminder-test.js (hunk 1 above)
Comment (Mickyfen17) [reply tests/unit/routes/reminders/reminder-test.js]: Removed

Comment (Mickyfen17) [reply tests/unit/routes/reminders/reminder-test.js]: @martensonbj I put used test but really meant unused....

**Pull request merged**